tests/Unit/Services/Gateways/BaseGatewayTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Tests\Unit\Services\Gateways;
+
+use Ebanx\Benjamin\Services\Gateways\BaseGateway;
+
+class BaseGatewayTest extends GatewayTestCase
+{
+    public function testInstalmentsArgentina()
```

Comment:
Move these to [CreditCardTest](https://github.com/ebanx/benjamin/blob/master/tests/Unit/Services/Gateways/CreditCardTest.php), instalments really don't belong here.